sites/fast-color-explorer/package.json

```diff
@@ -9,7 +9,7 @@
     "prettier": "prettier --config ../../.prettierrc --write \"**/*.{ts,tsx,html}\"",
     "prettier:diff": "prettier --config ../../.prettierrc \"**/*.{ts,tsx,html}\" --list-different",
     "start": "webpack-dev-server",
-    "test": "yarn eslint && yarn build",
```

Comment:
For packages that have / are *sites* I don't think removing `build` is safe. There is potential for code changes or dependency updates to affect whether these packages successfully build, and there is no CI process that validates these packages build if we remove `build` from `test`.